content-src/experiments/lockbox.yaml

```diff
@@ -0,0 +1,81 @@
+image_facebook: /static/images/experiments/notes/social/notes-fb.jpg
+video_url: https://www.youtube.com/embed/vaFvr_70yxw
+changelog_url: 'https://github.com/mozilla/notes/releases'
+contribute_url: 'https://github.com/mozilla/notes'
+bug_report_url: 'https://github.com/mozilla/notes/issues'
+discourse_url: 'https://discourse.mozilla-community.org/c/test-pilot/notes'
+privacy_notice_url: 'https://github.com/mozilla/notes/blob/master/docs/metrics.md'
+measurements:
+  - >
+    Notes will collect information about your use of the feature, such as how often you create notes
+    and how long you spend editing them, the number of characters you save in your notes, and whether
+    you use any editing features (e.g., bold or stylized text).
+  - >
+    Notes will also collect information about your interaction with the add-on, such as how you
+    access the feature, and what buttons you may click.
+xpi_url: 'https://testpilot.firefox.com/files/lockbox@mozilla.com/latest'
+addon_id: 'lockbox@mozilla.com'
+gradient_start: '#A665B4'
+gradient_stop: '#CF5858'
+details:
+  -
+    image: /static/images/experiments/notes/experiments_experimentdetail/detail1.jpg
+    copy: >
+      Tada!
+tour_steps:
+  -
+    image: /static/images/experiments/notes/experiments_experimenttourstep/notes-tour-1.jpg
+    copy: Notes adds a simple notepad to your browser’s sidebar.
+  -
+    image: /static/images/experiments/notes/experiments_experimenttourstep/notes-tour-2.jpg
+    copy: We’ve built some basic formatting options into Notes. With your feedback we will add more features.
+  -
+    image: /static/images/experiments/notes/experiments_experimenttourstep/notes-tour-3.jpg
+    copy: You can get back to the Notes experiment by clicking on the sidebar icon.
+  -
+    image: /static/images/experiments/notes/experiments_experimenttourstep/notes-tour-4.jpg
+    copy: You can always give us feedback or disable Notes from Test Pilot.
+contributors:
+  -
+    display_name: 'Udara Weerasinghege'
+    title: 'Engineering Intern'
+    avatar: /static/images/experiments/avatars/udara-w.jpg
+  -
+    display_name: 'Cedric Amaya'
+    title: 'Community Contributor'
+    avatar: /static/images/experiments/avatars/cedric-a.png
+  -
+    display_name: 'Ryan Feeley'
+    title: 'Senior Product/UX Designer'
+    avatar: /static/images/experiments/avatars/ryan-f.jpg
+  -
+    display_name: 'Remy Hubscher'
+    title: 'Software Engineer'
+    avatar: /static/images/experiments/avatars/remy-h.jpg
+  -
+    display_name: 'Vlad Filippov'
+    title: 'Senior Engineer'
+    avatar: /static/images/experiments/avatars/vlad-f.jpg
+  -
+    display_name: 'Peter Dehaan'
+    title: 'Firefox QA'
+    avatar: /static/images/experiments/avatars/peter.jpg
+  -
+    display_name: 'Emil Pasca'
+    title: 'Softvision QA'
+    avatar: /static/images/experiments/avatars/emil.jpg
+created: '2017-08-01T14:00:00Z'
+launch_date: '2017-08-01T14:00:00Z'
```

Comment:
Need to bump this date into the future before merging. Otherwise, this experiment will be published at our next deploy, temporary starting content and all. 

Might be better to keep this on another branch while it's in progress and get a PR together once the content is finalized

Reply:
@lmorchard: But this is set to `dev: true`, so it'd only be be published to testpilot.dev.mozaws.net, correct?

Reply:
Oh yeah, I missed the `dev: true` part. Probably okay, then.

Reply:
probably still good to get in the habit of having a future publish date in these types of prs